src/styles/_stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  padding: 0;
+  display: flex;
+  gap: 4px;
+  list-style: none;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url('../images/star.svg');
```

Comment:
Verify that the path '../images/star.svg' is correct and that the image file exists in the specified directory.